server/column_scope.cc

```diff
@@ -145,6 +145,52 @@ static CellValue extractVal(const CellValue & val, CellValue *)
     return val;
 }
 
+namespace {
+    // Made to init and hold a non POD array of dynamic size in a memory safe
+    // way.
+    template<typename T, size_t MAX_STACK_ENTRIES=4096/sizeof(T)>
```

Comment:
Exactement. Il faut que le nombre soit statique pour que ça compile, donc ici le nombre statique c'est 4kb / size of T.